automation/test.sh

```diff
@@ -436,7 +436,7 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
   elif [[ $TARGET =~ wg-s390x ]]; then
     label_filter='(wg-s390x)'
   elif [[ $TARGET =~ wg-arm64 ]]; then
-    label_filter='(wg-arm64 && !(ACPI,requires-two-schedulable-nodes,cpumodel))'
+    label_filter='(wg-arm64 && !(ACPI,requires-two-schedulable-nodes,cpumodel,requires-two-worker-nodes-with-cpu-manager))'
```

Comment:
Please leave comment in commit why this was added.

Reply:
Oh and for context https://github.com/kubevirt/kubevirt/issues/15066 is the reason.